sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -1648,7 +1648,7 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
 
   test("external sorting updates peak execution memory") {
     AccumulatorSuite.verifyPeakExecutionMemorySet(sparkContext, "external sort") {
-      sortTest()
+      sql("SELECT * FROM testData2 ORDER BY a ASC, b ASC").collect()
```

Comment:
This is actually a stricter test. What `verifyPeakExecutionMemorySet` does is assert that the peak memory is incremented in the code block. If we do many things in the code block then it's not guaranteed that a single sort like this one actually increments the peak memory.